packages/plugin-vue-jsx/index.d.ts

```diff
@@ -1,6 +1,12 @@
 import { Plugin } from 'vite'
 import { VueJSXPluginOptions } from '@vue/babel-plugin-jsx'
+import { FilterPattern } from '@rollup/pluginutils'
 
-declare function createPlugin(options?: VueJSXPluginOptions): Plugin
+declare interface CommonOptions {
+    include?: FilterPattern;
+    exclude?: FilterPattern;
+}
```

Comment:
```suggestion
declare interface CommonOptions {
  include?: FilterPattern
  exclude?: FilterPattern
}
```

/cc @Shinigami92 our linter does not catch this?

Reply:
I can look into that, but Evan said, that the `gitHooks pre-commit lint-staged` should handle such things
Seems it doesn't + users can just ignore/bypass it :confused: